### ISSUE TITLE
not consider numDecimals = 0

### ISSUE THREAD

**Issue by leohgbs** [schowdhuri/round-half-even#1]

**Comment by schowdhuri** [comment]
@leohgbs Do you see an error when you set `numDecimals` to 0? Feel free to play around with [the demo.](https://rawgit.com/schowdhuri/round-half-even/master/demo/index.html)

**Comment by leohgbs** [comment]
```
roundHalfEven(255.552,0)
demo: 255
answer: 256
```

my solution
```
 if (numDecimals === 0) {
        return roundHalfEven(value/10, 1)*10
 }
```

**Comment by schowdhuri** [comment]
> ```
> roundHalfEven(255.552,0)
> demo: 255
> answer: 256
> ```
> 
> my solution
> 
> ```
>  if (numDecimals === 0) {
>         return roundHalfEven(value/10, 1)*10
>  }
> ```

@leohgbs Thanks for catching that! Love your fix! Do you wanna send a PR?

**Comment by schowdhuri** [comment]
Fixed in `v1.2.1`
